scripts/update-omni-kpis.sh

```
@@ -0,0 +1,70 @@
+
+if [ ! -f "$DATA_FILE" ]; then
+  echo "Error: $DATA_FILE not found"
+  exit 1
+fi
+
+# 嘗試從 Ollama Cloud 取得真實資料（若 API Key 存在）
+ollama_status="unknown"
+if [ -n "$OLLAMA_KEY" ]; then
+  response=$(curl -s --max-time 10 "$OLLAMA_URL" \
+    -H "Authorization: Bearer $OLLAMA_KEY" 2>/dev/null)
+  if [ $? -eq 0 ] && [ -n "$response" ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)